src/js/ctlevent.js

```diff
@@ -22,13 +22,23 @@ var propertiesString = function(properties) {
     return propString;
 };
 
+let compareDates = (start, end) => {
+    let startStr = new Date(start.toLocaleDateString());
+    let endStr = new Date (end.toLocaleDateString());
+
+    return endStr > startStr;
+};
+
 var CTLEvent = function(event) {
     this.id = event.guid;
     this.title = event.summary;
 
     this.startDate = 'start' in event ? CTLEventUtils.strToDate(event.start.datetime) : '';
     this.endDate = 'end' in event ? CTLEventUtils.strToDate(event.end.datetime) : '';
-    this.multiDay = this.startDate && this.endDate ? this.endDate.getDate() > this.startDate.getDate() : false;
+
+    if(this.startDate && this.endDate ) {
```

Comment:
This if statement should be changed to `if (this.startDate && this.endDate) {`, but that's just a style thing. Otherwise I'll just go ahead and merge this.